USAGE.md

```diff
@@ -187,6 +187,12 @@ For example, if we defined the following schema definition in a file named `mypr
         Unique identifier of the widget.
 ```
 
+Multiple directory targets can also be provided:
+
+```
+$ python scripts/generator.py --include ../myproject/custom-fields-A/  ../myproject/custom-fields-B --out ../myproject/out/
```

Comment:
For readability I would put one per line here instead, to make sure the proper way to do it stands out

```suggestion
$ python scripts/generator.py \
    --include ../myproject/custom-fields-A/  ../myproject/custom-fields-B \
    --out ../myproject/out/
```